comid/tdx-profile/teedigest.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+// nolint:dupl
+package tdx
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/corim/encoding"
+)
+
+type Digests = comid.Digests
+
+// TeeDigest, holds the digests. Allowed values are an array of digests OR
+// a digest expression
+type TeeDigest struct {
```

Comment:
Since this can hold multiple digets, shouldn't this be called `TeeDigests`?

Reply:
the name is aligned with how it is documented in the profile, so I would retain it to `TeeDigest` for now!